src/transformers/modeling_t5.py

```diff
@@ -503,10 +503,12 @@ def _init_weights(self, module):
 
 
 class T5Stack(T5PreTrainedModel):
-    def __init__(self, config):
+    def __init__(self, config, embed_tokens=None):
```

Comment:
This is an important change: To make T5 API similar to Bart, the encoder and decoder should be able to encode `input_ids` directly. @thomwolf @craffel